tests/DCPS/Compiler/anonymous_types/anonymous_types.mpc

```diff
@@ -0,0 +1,7 @@
+project: dcps {
+  idlflags += -as
```

Comment:
This should move to a base project so all projects can inherit the setting.

Reply:
These lines were a workaround until https://github.com/DOCGroup/ACE_TAO/pull/1135 was merged. Since it was merged both lines can be removed if using DOCGroup master.

Reply:
We should still have `-as` for tao_idl since some versions will need it.